planner/core/exhaust_physical_plans.go

```diff
@@ -2266,20 +2266,20 @@ func (la *LogicalAggregation) tryToGetMppHashAggs(prop *property.PhysicalPropert
 			}
 		}
 		// TODO: permute various partition columns from group-by columns
-		// 1-phase
+		// 1-phase agg
 		childProp := &property.PhysicalProperty{TaskTp: property.MppTaskType, ExpectedCnt: math.MaxFloat64, PartitionTp: property.HashType, PartitionCols: partitionCols, CanAddEnforcer: true}
 		agg := NewPhysicalHashAgg(la, la.stats.ScaleByExpectCnt(prop.ExpectedCnt), childProp)
 		agg.SetSchema(la.schema.Clone())
 		agg.MppRunMode = Mpp1Phase
 		hashAggs = append(hashAggs, agg)
-		// 2-phase
+		// 2-phase agg
 		childProp = &property.PhysicalProperty{TaskTp: property.MppTaskType, ExpectedCnt: math.MaxFloat64, PartitionTp: property.AnyType}
 		agg = NewPhysicalHashAgg(la, la.stats.ScaleByExpectCnt(prop.ExpectedCnt), childProp)
 		agg.SetSchema(la.schema.Clone())
 		agg.MppRunMode = Mpp2Phase
 		agg.PartitionCols = partitionCols
 		hashAggs = append(hashAggs, agg)
-
+		// agg runs on TiDB
```

Comment:
//aggregation that partially runs on TiDB

Reply:
updated.